x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardChangesAction.java

```diff
@@ -214,56 +214,54 @@ public String toString() {
 
     public static final class Response extends ActionResponse {
 
-        private long mappingVersion;
+        private final long mappingVersion;
```

Comment:
That is one very unusual field ordering :)
Could this one also be converted to a record?